src/codeflare_sdk/utils/demos.py

```diff
@@ -0,0 +1,27 @@
+import pathlib
+import shutil
+
+package_dir = pathlib.Path(__file__).parent.parent.resolve()
+demo_dir = f"{package_dir}/demo-notebooks"
+
+
+def clone_demo_nbs(dir: str = "./demo-notebooks", overwrite: bool = False):
```

Comment:
What about `copy_demo_notebooks`? Clone gives too much impression it's a Git operation.